src/views/createEditCommunity/create.tsx

```diff
@@ -145,31 +143,36 @@ function CreateCommunityScreen() {
                         setSubmittingCover(false);
                         setSubmittingProfile(false);
                     })
-                    .catch((error) => console.log(error));
+                    .catch((error) => Sentry.Native.captureException(error));
```

Comment:
No, we don't want to catch this error here. Likely, it was forgotten.
Same for `cancelablePromise` above.

Reply:
Removed from both places, but don't you think it's important to catch errors from either submitCommunity and uploadImages functions execution since we know this is our biggest source of problem within community creation?

Reply:
The error thrown here is from https://github.com/impactMarket/mobile-app/pull/183/files/d8547137376122b9441abfcd395b15b2fc361d36..b32f65664a7cfbcd19823fc0650ffecba9d36900#diff-cb80ad01a00b31b841eafa73a55a1b00a07d2672b5d1d89bb3a483df1b8f1cc2R56
image uploads handle its own error.

Reply:
Got it.